src/praisonai/praisonai/cli/app.py

```
@@ -35,6 +35,36 @@ def _setup_langfuse_observability(*, verbose: bool = False) -> None:
+        # Create LangextractSink
+        sink = LangextractSink(config=config)
+        
+        # Set up action-level trace emitter
+        emitter = TraceEmitter(sink=sink, enabled=True)
+        set_default_emitter(emitter)
+        
+    except ImportError:
+        # Gracefully degrade if langextract not installed
+        if verbose:
+            typer.echo("Warning: langextract is not installed. Install with: pip install 'praisonai[langextract]'", err=True)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)